File: Examples/FxSocket.Sftp/Windows/CS/SftpDesktopClientDemo/NewNameForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace FxSocketSamples
{
    public partial class NewNameForm : Form
    {
        public NewNameForm()
        {
            InitializeComponent();
        }

        public NewNameForm(string title) : this()
        {
            this.Text = title;
        }

        /// <summary>
        /// Gets the new name.
        /// </summary>
        public string NewName
        {
            get
            {
                return txtNewName.Text;
            }
            set { txtNewName.Text = value; }
        }
    }
}

File: Examples/FxSocket.Sftp/Windows/VB/SftpDesktopClientDemo/NewNameForm.vb
Imports System.ComponentModel
Imports System.Text

Namespace FxSocketSamples
	Partial Public Class NewNameForm
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Public Sub New(ByVal title As String)
			Me.New()
			Me.Text = title
		End Sub

		''' <summary>
		''' Gets the new name.
		''' </summary>
		Public Property NewName() As String
			Get
				Return txtNewName.Text
			End Get
			Set(ByVal value As String)
				txtNewName.Text = value
			End Set
		End Property
	End Class
End Namespace
